site/gatsby-site/src/contexts/userContext/UserContextProvider.js

```diff
@@ -18,11 +18,15 @@ import { UserContext } from './UserContext';
 
 export const UserContextProvider = ({ children }) => {
   const [loading, setLoading] = useState(false);
+
   const [user, setUser] = useState();
 
   const [type, setType] = useState();
+
   const [mongoUserKey, setMongoUserKey] = useState(null);
 
+  const apiKey = window.localStorage.getItem('mongoUserKey');
```

Comment:
I believe this line is breaking the build since the `window` object will not be defined during SSR. I think this can be solved by assigning it to a placeholder during SSR.